### PR TITLE
fix the backend reviews/rating to make sure that am getting username …

### PR DESCRIPTION
**Pull request by JosephNgabo**
…to display on the feedbacks

#### What does this PR do?
This PR gets the username to be displayed on the feedback/reviews added to make sure that every user who adds review thir username will be displayed.

### CODE REVIEW

File: src/services/reviews.service.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+import User from '../database/models/user.model';
 
 async function getReviews(productId) {
-  const reviews = await Reviews.findAll({ where: { productId } });
+  const reviews = await Reviews.findAll({
+    where: { productId },
+    include: [
+      {
+        model: User,
+        attributes: ['username'], // Specify the field(s) you want to retrieve
```
Comment (munezerobagira) [inline src/services/reviews.service.js]: you can include the user profile as well,  if like first name and last name  and profile image is possible

**Review by munezerobagira — CHANGES_REQUESTED** [review]

File: src/services/reviews.service.js (hunk 1 above)
Comment (JosephNgabo) [reply src/services/reviews.service.js]: > you can include the user profile as well, if he like first name and last name and profile image is possible
Screen Shot 2023-06-23 at 10 34 53
for the backend there is no option for adding user profile image and also names are displayed all at the same time as `names`

Comment (munezerobagira) [reply src/services/reviews.service.js]: - if no profile image, We need to have name only,  not birthday and telephone

File: src/controllers/review.controller.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ const getReviews = (req, res) => {
 };
 
 const addReview = (req, res) => {
+  // const userId = req
```
Comment (munezerobagira) [inline src/controllers/review.controller.js:15]:
```suggestion
```

**Review by munezerobagira — CHANGES_REQUESTED** [review]

**Review by Edcreation — APPROVED** [review]

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**